core/amber/src/main/scala/edu/uci/ics/texera/web/model/collab/event/CollabWebSocketEvent.scala

```diff
@@ -0,0 +1,17 @@
+package edu.uci.ics.texera.web.model.collab.event
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type
+import com.fasterxml.jackson.annotation.{JsonSubTypes, JsonTypeInfo}
+import edu.uci.ics.texera.web.model.collab.response.HeartBeatResponse
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+@JsonSubTypes(
+  Array(
+    new Type(value = classOf[CommandEvent]),
+    new Type(value = classOf[LockGrantedEvent]),
+    new Type(value = classOf[ReleaseLockEvent]),
+    new Type(value = classOf[LockRejectedEvent]),
+    new Type(value = classOf[ReloadWorkflowEvent]),
+    new Type(value = classOf[HeartBeatResponse])
+  )
+)
+trait CollabWebSocketEvent {}
```

Comment:
If most subclasses are just empty case classes, we can just put them all in a single file.

Reply:
I'm following `TexeraWebSocket`'s conventions here.